lib/ae_page_objects/elements/collection.rb

```diff
@@ -56,6 +56,11 @@ def configure(options)
       @item_locator = options.delete(:item_locator) || default_item_locator
     end
 
+    def options
+      evaled_locator = eval_locator(@item_locator)
```

Comment:
In the previous changeset, you were memoizing the result of this method (https://github.com/appfolio/ae_page_objects/commit/6895134394e256d8d475eaa94bb311df8fecd1cd#diff-4edadc6ec811dffb9a89b8cc6d88feccR60). Was that intentionally removed?